2015/about.html

```diff
@@ -101,7 +101,7 @@ <h2 class="text-center top-space"></h2>
         <hr>
         <h3>Request Our White Papers</h3>Cornerstone is professionally committed to creating effective, research-based solutions to our clients’ need to identify and prepare top talent to become effective leaders and managers. Our system is based on peer-reviewed, published research and on basic statistical principles universally recognized as tools for studying and evaluating this type of material. This approach protects organizations from investing in programs that appear effective but do not pass the test of time.<br>
         <br>
-        To learn more about why we do what we do and how we are different from (and more effective than) traditional assessment centers, <a href="/home/contact">click here</a> to request a copy of our white papers:<br>
+        To learn more about why we do what we do and how we are different from (and more effective than) traditional assessment centers, <a href="/home/contact.html">click here</a> to request a copy of our white papers:<br>
```

Comment:
This link doesn't work because `/home` is not valid
I think it should just be `contact.html` and then it will fall under the 2015 folder or be like what you did on :42